specs-go/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/container-orchestrated-devices/container-device-interface/specs-go/v0.0.0
```

Comment:
This version at the end is surprising. Is it needed?

Reply:
No, I think it is a mistake to have it there.

Reply:
fixed. thank you for pointing out.